test/distributed/test_c10d.py

```diff
@@ -4649,6 +4649,29 @@ def test_gloo_barrier_device_ids(self):
         with self.assertRaisesRegex(RuntimeError, "device_ids not supported"):
             c10d.barrier(device_ids=[self.rank])
 
+    def test_distributed_debug_mode(self):
+        # Default should be info
+        default_debug_mode = dist.get_debug_mode()
+        self.assertEqual(default_debug_mode, dist.DistributedDebugMode.INFO)
+        mapping = {
+            "OFF": dist.DistributedDebugMode.OFF,
+            "INFO": dist.DistributedDebugMode.INFO,
+            "DETAIL": dist.DistributedDebugMode.DETAIL,
+        }
+        invalid_debug_modes = [None, "foo", -1]
+        for invalid_mode in invalid_debug_modes:
+            mapping[invalid_mode] = dist.DistributedDebugMode.INFO
```

Comment:
Seems like these should raise an error instead of defaulting to INFO. I guess only `None` should default to INFO, but rest (foo, -1) should raise an error.

Reply:
This sounds good, although I guess I thought a benefit of the current approach is we'd default to something sensible in these cases, though it might be better to be explicit and error out

Reply:
Actually, the `None` doesn't really make sense here since we convert it to string, and "None" should also be an invalid value.